circle.yml

```diff
@@ -11,6 +11,7 @@ test:
     - cd $IMPORTPATH && go test -v -race $(go list ./... | grep -v /vendor/)
     - cd $IMPORTPATH && go vet $(go list ./... | grep -v /vendor/)
     - cd $IMPORTPATH && scripts/end2end_test.sh
+    - cd $IMPORTPATH && go test -run='^$' -bench=. $(go list ./... | grep -v /vendor/)
```

Comment:
end2end_test tears down the docker containers when it finishes so all of the benchmark tests that have an external dependency will fail